src/mapping.ts

```diff
@@ -199,10 +215,13 @@ function assignProjectToPlatform(sender: string, platformId : string, projectId:
   if (project == null) return
 
   /* check if sender is owner OR admin of platform */
-  if (platform.owner != sender && !UserPlatform.load(buildMappingTableId(sender, platformId))) return
+  if (platform.owner != sender && !AdminPlatform.load(buildMappingTableId(sender, platformId))) return
 
-  project.platform = platformId
-  project.save()
+  /* create platformProject mapping */
+  let platformProject = new PlatformProject(buildMappingTableId(projectId, platformId))
```

Comment:
check order of platformId and projectId, according what we talked, we need to change the order

Reply:
good catch